pgcli/packages/sqlcompletion.py

```diff
@@ -291,7 +291,9 @@ def suggest_based_on_last_token(token, text_before_cursor, full_text,
             ('copy', 'from', 'update', 'into', 'describe', 'truncate')):
 
         schema = (identifier and identifier.get_parent_name()) or None
-
+        # If schema name is unquoted, lower-case it
+        if schema and identifier and identifier.value[0] != '"':
```

Comment:
You're right that the identifier check was redundant, so I've removed that, but we do need to use identifier.value for the ", as it's been stripped from schema.